### PR TITLE
[Trivial] Replace "doesn't" with "does not" in error message

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

File: libsolidity/analysis/TypeChecker.cpp [hunk 1]
```diff
@@ -1336,7 +1336,7 @@ bool TypeChecker::visit(Conditional const& _conditional)
 					_conditional.location(),
 					"True expression's type " +
 					trueType->toString() +
-					" doesn't match false expression's type " +
+					" does not match false expression's type " +
```
Comment (ekpyron) [inline libsolidity/analysis/TypeChecker.cpp:1339]: Actually, I think the ``'s``es aren't that idiomatic either here actually... if you want, you could change it further to:
``The type of the true expression ... does not match the type of the false expression ... .``
But both is fine.

**Review by ekpyron — DISMISSED** [review]

File: libsolidity/analysis/TypeChecker.cpp (hunk 1 above)
Comment (Marenz) [reply libsolidity/analysis/TypeChecker.cpp:1339]: will do

**Review by ekpyron — DISMISSED** [review]
Hm... seeing this in the tests now makes me think: maybe not perfect after all :-).
``The type "bool" of the true expression does not match the type "uint8" of the false expression.``
might be nicer -  ``The type of the true expression bool`` reads a bit like ``bool`` is supposed to be the expression, not it's type... I guess that's why it *was* ``The true expression's type`` before :-).

But still fine either way.

**Comment by Marenz** [comment]
reverted it

**Review by chriseth — APPROVED** [review]

**Pull request merged**